src/ApplicationFactory.cc

```diff
@@ -0,0 +1,54 @@
+/*
+ * ApplicationBase.cc
+ *
+ *  Created on: Nov 11, 2016
+ *      Author: Martin Hierholzer
+ */
```

Comment:
Wrong header. It's not only a copy+paste error, but it should now be in the SPDX format.

Reply:
fixed